pkg/rehearse/jobs.go

```diff
@@ -628,6 +628,31 @@ func (e *Executor) submitRehearsals() ([]*pjapi.ProwJob, error) {
 	var pjs []*pjapi.ProwJob
 
 	for _, job := range e.presubmits {
```

Comment:
Could we get a function that takes `[]args` and spits out post-processed args, so that we can unit test it?